index.bs

```diff
@@ -5747,9 +5745,8 @@ in the constructor.
 
  1. Let |steps| be the following steps, given |chunk|:
   1. Return ? [$GetV$](|chunk|, "`byteLength`").
- 1. Let |F| be ! [$CreateBuiltinFunction$](|steps|, « », |globalObject|'s [=relevant Realm=]).
- 1. Perform ! [$SetFunctionName$](|F|, "`size`").
- 1. Perform ! [$SetFunctionLength$](|F|, 1).
+ 1. Let |F| be ! [$CreateBuiltinFunction$](|steps|, 1, "`size`", « », |globalObject|'s [=relevant
+    Realm=]).
```

Comment:
Actually sorry, this is technically a normative change as it changes the enumeration order of `name` vs `length`, but Chrome and Firefox both already conform to this change (`length` comes before `name).

Reply:
Thanks for the followup, that's interesting. I think we don't need to go through the normative process for this.